example.py

```diff
@@ -8,32 +8,32 @@
 
 twitch_miner = TwitchChannelPointsMiner(
     username="your-twitch-username",
-    make_predictions=True,              # If you want to Bet / Make prediction | The browser will never start
-    follow_raid=True,                   # Follow raid to obtain more points
-    watch_streak=True,                  # If a streamer go online change the priotiry of streamers array and catch the watch screak. Issue #11
-    drops_events=True,                  # If you want to auto claim game drops from Twitch inventory Issue #21
-    claim_drops_startup=False,          # If you want to auto claim all drops from Twitch inventory on startup
+    make_predictions=True,  # If you want to Bet / Make prediction | The browser will never start
```

Comment:
About this I need that the example.py contains the same indention for readme (just for make it clear to read).
So, please copy and paste the current example.py from master and reset the indentation.